src/Server/Workspace.php

```diff
@@ -200,4 +214,75 @@ public function xdependencies(): array
         }
         return $dependencyReferences;
     }
+
+    /**
+     * Fires when client changes settings in the client
+     *
+     * The default paramter type is Options but it also accepts different types
+     * which will be transformed on demand.
+     *
+     * Currently only the vscode format is supported
```

Comment:
What does this mean? Where is the problem in saying the PHP LS reads settings under the `php` key?